langstream-runtime/langstream-runtime-impl/src/main/python/langstream/util.py

```diff
@@ -29,31 +29,31 @@ def __init__(
         self,
         value,
         key=None,
+        headers: List[Tuple[str, Any]] = None,
```

Comment:
Small breaking change here for consistency.
Only affects if you were using positional args and not named ones.